test/verify/check-metrics

```diff
@@ -1261,6 +1265,23 @@ BEGIN {{
         b.wait_text("[aria-label='Network usage'] [data-interface='cockpittest1'] td[data-label='In']", "0")
         b.wait_text("[aria-label='Network usage'] [data-interface='cockpittest1'] td[data-label='Out']", "0")
 
+        b.click("[aria-label='Network usage'] [data-interface='lo'] button")
+        b.enter_page("/network")
+        b.wait_visible("#network-interface-name:contains('lo')")
+        if "ubuntu-2204" in m.image or "centos-8-stream" in m.image or "rhel-8-9" in m.image:
```

Comment:
```suggestion
        if "ubuntu" in m.image or "centos-8-stream" in m.image or "rhel-8" in m.image:
```

This will affect all RHEL images, and most certainly also ubuntu-stable, as they use networkd by default.

Reply:
Changed.

Reply:
Sorry, [you were right about ubuntu-stable](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19339-20230919-043034-85032c5e-ubuntu-stable-other/log.html#167). Fixed.